src/main/java/billennium/tests/service/user/ExecutingQuizServiceImpl.java

```diff
@@ -40,7 +41,7 @@ public void saveUser(String email) {
         return userRepository.findById(userId);
     }
 
-    private List<QuizDefinition> getAllQuiz() {
+    private List<QuizDefinition> getAllQuiz() { // TODO unused method
```

Comment:
Za to aż mi wstyd niby projekt brudnopis póki co ale mam skłonność lekko do tego że nie usuwam wszystkiego.